content/about/board/2021-07-21.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+show_meta: false
+title: 'Terse Notes from Board Meeting on 2021-07-21'
+---
+
+### Date and Time of Meeting
+
+2021-07-21, 7 p.m - 8 p.m. CEST / 12 a.m. - 1 p.m. CDT
+Call to order 19:04 CEST - Adjourned 19:52 CEST
+
+### Role Call
+
+#### Directors Present
+
+- Danese Cooper (Chair)
+- Jacob Green
+- Johannes Tigges (Assistant Secretary)
+- Isabel Drost-Fromm (President)
+- Cedric Williams (Treasurer)
+- Daniel Izquierdo
+
+
+#### Directors Absent
+
+- Georg Grütter (Vice President)
+- Russel Rutledge (Secretary)
```

Comment:
```suggestion
- Russell Rutledge (Secretary)
```

Reply:
Oh, sorry, I forgot the second l again! Thanks for adding it. xD